cogboard-webapp/src/components/widgets/types/LogViewerWidget/Toolbar/DateRangePicker/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import MomentUtils from '@date-io/moment';
+import { MuiPickersUtilsProvider, DateTimePicker } from '@material-ui/pickers';
+import ToolbarGroup from '../ToolbarGroup';
+
+export default function DateRangePicker() {
+  return (
+    <ToolbarGroup title="Date span">
+      <MuiPickersUtilsProvider utils={MomentUtils}>
+        <DateTimePicker
+          label="Begin"
+          format="hh:mm DD/MM/YY"
+          onChange={() => console.log('TODO')}
+        />
+        <DateTimePicker
+          label="End"
+          format="hh:mm DD/MM/YY"
+          onChange={() => console.log('TODO')}
+        />
+      </MuiPickersUtilsProvider>
+    </ToolbarGroup>
+  );
+}
```

Comment:
Please use arrow function syntax
```suggestion
const DateRangePicker = () => (
    <ToolbarGroup title="Date span">
      <MuiPickersUtilsProvider utils={MomentUtils}>
        <DateTimePicker
          label="Begin"
          format="hh:mm DD/MM/YY"
          onChange={() => console.log('TODO')}
        />
        <DateTimePicker
          label="End"
          format="hh:mm DD/MM/YY"
          onChange={() => console.log('TODO')}
        />
      </MuiPickersUtilsProvider>
    </ToolbarGroup>
  );
```